docs/_static/css/custom.css

```diff
@@ -1,21 +1,39 @@
 /* ReadTheDocs theme colors */
 
-
 .wy-nav-top { background-color: #f07e44 }
+.wy-nav-content {
+    max-width: 1200px;
```

Comment:
What does this do?

Reply:
See the Slack post in #docs... the wider layout...

Reply:
can you enable this branch on readthedocs and link it from the PR description please?

Reply:
I did this before: https://icb-scanpy.readthedocs-hosted.com/en/docs-for-1.4.5/